helm-charts/aerospike-cluster/templates/_helpers.tpl

```diff
@@ -25,8 +25,12 @@ If release name contains chart name it will be used as a full name.
 {{- end -}}
 
 {{- define "aerospike-cluster.commonName" -}}
+{{- if .Values.commonName -}}
+{{- .Values.commonName -}}
```

Comment:
Can you add a commented out entry in `values.yaml` for `commonName` configuration?
and also document the configuration in README file?

Reply:
Hello @spkesan I made the changes you requested in another commit; if it is ok for you, I will squash everything before the merge.